Ginger/Ginger/AutomatePageLib/AddActionMenu/SharedRepositoryLib/RepositoryItemUsagePage.xaml.cs

```diff
@@ -357,7 +357,7 @@ private async void UpdateAllButton_Click(object sender, RoutedEventArgs e)
             {
                  StartProcessingIcon();
          
-                Parallel.ForEach(RepoItemUsages, usage =>
+                foreach(var usage in RepoItemUsages)
```

Comment:
What is the issue with parallel For each ? 
Check if same issue is applicable also for SaveAllBizFlowsButton_Click